plugins/grpc-ecosystem/gateway/source.yaml

```diff
@@ -1,3 +1,4 @@
 source:
-  goproxy:
-    name: github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway
+  github:
+    owner: grpc-ecosystem
+    repository: grpc-gateway
```

Comment:
This should fix https://github.com/bufbuild/plugins/actions/runs/3944373335/jobs/6750199838